src/app/find-help/transport-search/transport-search-form/transport-search-form.component.html

```diff
@@ -1,29 +1,33 @@
 <form (ngSubmit)="search.emit(data)" #form="ngForm">
   <div class="container">
     <div class="row">
-      <div class="col-md-3">
+      <div class="col-md-4 col-lg-3">
         <app-cities-search
           [(ngModel)]="data.origin"
-          name="origin"
+          name="location"
```

Comment:
Jaki jest powód do zmiany tego? To samo pytanie poniżej.

Reply:
powód raczej z rozpędu, dzięki